src/odemis/gui/comp/overlay/view.py

```diff
@@ -343,6 +343,14 @@ def draw(self, ctx):
                     margin_r = 5
                     margin_h += (self._label.text_size[1] + margin_r)
 
+            # Display physical position at the top
+            pos = units.readable_str(self._p_pos, "m", sig=6)
+            text = f"Position: {pos}"
+            view_pos = self.view_width - margin_w, self.view_height - margin_h
+            self._label.pos = Vec(view_pos[0], view_pos[1])
+            self._label.text = text
+            self._label.align = wx.ALIGN_RIGHT
```

Comment:
This line is actually also always the same. So could you move it with the colour, line 319? (and also drop the line 338).